app/models/user.rb

```diff
@@ -153,7 +156,7 @@ def clear_call_list
   end
 
   def admin?
-    role == :admin
+    role == 'admin'
```

Comment:
This might be redundant to predicate stuff provided by the new enum library -- wonder if we can nuke it?